warnings.rb

```diff
@@ -0,0 +1,10 @@
+
+a = { "hello" => "world" }
```

Comment:
_[farcy v0.1b](https://github.com/appfolio/farcy)_

* Lint/UselessAssignment: Useless assignment to variable - `a`.
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.